kaolin/datasets/shapenet.py

```diff
@@ -193,6 +160,85 @@ def __getitem__(self, index):
         return {'data': data, 'attributes': attributes}
 
 
+class ShapeNet(KaolinDataset):
+    r"""ShapeNet Dataset class for meshes.
+    Args:
+        root (str): path to ShapeNet root directory
+        categories (list): List of categories to load from ShapeNet. This list may
+                           contain synset ids, class label names (for ShapeNetCore classes),
+                           or a combination of both.
+        train (bool): is using training split (default: True)
```

Comment:
```
split to use (train if True, test otherwise) (default:True)
```

Reply:
Fixed, I used [this](https://pytorch.org/docs/stable/torchvision/datasets.html) for reference